go/libraries/doltcore/mvdata/table_data_loc.go

```diff
@@ -239,9 +239,13 @@ func (te *tableEditorWriteCloser) WriteRow(ctx context.Context, r row.Row) error
 	_ = atomic.AddInt64(&te.gcOps, 1)
 
 	if te.insertOnly {
-		_ = atomic.AddInt64(&te.statOps, 1)
-		te.stats.Additions++
-		return te.tableEditor.InsertRow(ctx, r)
+		if err := te.tableEditor.InsertRow(ctx, r); err != nil {
```

Comment:
Don't like this if / else style for error handling. It's not actually any more compact and is difficult to scan. Just put if err != nil on its own line.

Reply:
Same comment elsewhere in this file.